langs/ru.h

```diff
@@ -261,3 +318,30 @@ msgstr("Невозможно найти выбранный файл или вы
 
 msgid(CLEAR_HISTORY)
 msgstr("Очистить историю")
+
+msgid(AUTO_STARTUP)
+msgstr("Запускать при загрузке системы")
+
+msgid(THEME)
+msgstr("Тема")
+
+msgid(THEME_DEFAULT)
+msgstr("По умолчанию")
+
+msgid(THEME_LIGHT)
+msgstr("Светлая")
+
+msgid(THEME_DARK)
+msgstr("Тёмная")
+
+msgid(THEME_HIGHCONTRAST)
+msgstr("Высококонтрастная")
+
+msgid(THEME_CUSTOM)
+msgstr("Своя (см. документацию)")
+
+msgid(THEME_ZENBURN)
+msgstr("Zenburn")
```

Comment:
I just forgot to delete this line. I suppose we don't need to translate Zenburn since it's a [proper noun](http://kippura.org/zenburnpage).